__tests__/app.test.js

```diff
@@ -102,6 +103,54 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of all the comments according to the article Id provided", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            body: expect.any(String),
+            votes: expect.any(Number),
+            author: expect.any(String),
+            article_id: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("200: Responds with an array of all the comments default sorted by most recent first", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("400: sends an appropriate status and error message when given an invalid format id", () => {
+    return request(app)
+      .get("/api/articles/meltedbrain/comments")
```

Comment:
meltedbrain!!! 

I understand, but maybe change this to `error` or `invalidEndpoint` - don't forget this is a portfolio piece :)